README.md

```diff
@@ -0,0 +1,255 @@
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+## 🪵 Documentation <a name="key-features"></a>
+
+- [ API Documentation Here](https://documenter.getpostman.com/view/26957552/2s9XxsVGHT)
+
+# 📖 Appoint With A Doctor Back-End <a name="about-project"></a>
+
+The Appoint Doctor Project is a user-friendly web app for booking doctor appointments. It offers secure login using Devise-jwt and supports multiple user roles (patient, doctor, admin). Admins can add/delete doctors. Users can schedule/cancel appointments and view their appointment list. The app simplifies the process of finding and managing medical appointments online.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><p> <b>Ruby: </b> The primary programming language for the project.</p></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Database</summary>
+ <ul>
+    <li><a href="https://www.postgresql.org/download/windows/">Postgresql</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Ruby On Rails</summary>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Authenticate User Login using Devise-jwt**
+- **Add Doctor**
+- **Delete Doctor**
+- **List of all Users**
+- **List of all Doctors**
+- **List of all Patients**
+- **Make Appointment with Doctor**
+- **My Appointment**
+- **Cancel Appointment**
+- **My Appointment List**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Mac or PC
+- Code Editor (ex: Vs Code)
+- Terminal
+- Postgresql
+- Node.js
+- git
+- Install [ruby](https://www.ruby-lang.org/en/documentation/installation/) on your computer
+- Install [rails](https://www.tutorialspoint.com/ruby-on-rails/rails-installation.htm#) on your computer
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd your-desired-folder
+  https://github.com/Tracy-miranja/appoint-doctor-backend
+```
+
+### Install
+
+- Install this project with:
+
+- Linters
+- Rubocop
+- Ruby On Rails
+
+- Follow the step:
+
+```sh
+cd appoint-doctor-backend
+npm install
+bundle install
+rails s
+```
+
+### Usage
+
+- Run the app with `rails s` in the terminal from the root folder of the project.
+
+### Test
+
+```sh
+  rspec spec/filename.rb
+```
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

Reply:
Dear @Meltrust Thanks for your valuable suggestion, We will add everything that you suggested.🎉😍